src/Symfony/Component/Routing/Tests/Generator/UrlGeneratorTest.php

```diff
@@ -863,10 +869,10 @@ public function testLookRoundRequirementsInPath($expected, $path, $requirement)
 
     public function provideLookAroundRequirementsInPath()
     {
-        yield ['/app.php/a/b/b%28ar/c/d/e', '/{foo}/b(ar/{baz}', '.+(?=/b\\(ar/)'];
-        yield ['/app.php/a/b/bar/c/d/e', '/{foo}/bar/{baz}', '.+(?!$)'];
-        yield ['/app.php/bar/a/b/bam/c/d/e', '/bar/{foo}/bam/{baz}', '(?<=/bar/).+'];
-        yield ['/app.php/bar/a/b/bam/c/d/e', '/bar/{foo}/bam/{baz}', '(?<!^).+'];
+        yield ['/app.php/a%2Fb/b(ar/c%2Fd%2Fe', '/{foo}/b(ar/{baz}', '.+(?=/b\\(ar/)'];
+        yield ['/app.php/a%2Fb/bar/c%2Fd%2Fe', '/{foo}/bar/{baz}', '.+(?!$)'];
+        yield ['/app.php/bar/a%2Fb/bam/c%2Fd%2Fe', '/bar/{foo}/bam/{baz}', '(?<=/bar/).+'];
+        yield ['/app.php/bar/a%2Fb/bam/c%2Fd%2Fe', '/bar/{foo}/bam/{baz}', '(?<!^).+'];
```

Comment:
are these changes required?
if not, better add new lines in the provider and keep existing lines as is.

Reply:
Given the current implementation these changes are required. I think this will change for the final implementation to maintain some level of BC and will adapt to keep changes to existing tests to a minimum. For now that outlines the implications.